sql/src/test/java/io/crate/integrationtests/RenameTableIntegrationTest.java

```diff
@@ -84,6 +84,19 @@ public void testRenameClosedTable() {
     }
 
     @Test
+    public void testCloseTableWithCustomSchema() {
```

Comment:
I don't think we need another integration test for this, but maybe a unit test for Schemas.getNewCurrentSchemas. However 
Also, please either modify the above test (`testRenameClosedTable`) to include a custom schema table to rename or add a new test `testRenameCustomSchemaClosedTable` so we cover both.